Update data-generator.py

duy-deliver opened the pull request:
- Code python for creating fake Database : Insurance , Patient, Disease, Symptom , DIagnosis

duy-deliver committed:
Update data-generator.py

- Code python for creating fake Database : Insurance , Patient, Disease, Symptom , DIagnosis

huyanh0720 reviewed (commented):
- Insurance_ID from Patient table should start with I not with P.

duy-deliver committed:
Update data-generator.py

+ Edit row Patient from 3001 -> 3000
+ Change Insurance_ID in Patient
+ Fix bug Symptom Name

huyanh0720 reviewed (approved):
Looking Good.
